src/RA_httpthread.cpp

```diff
@@ -256,9 +258,10 @@ void RAWeb::SetUserAgentString()
     #define NTSTATUS long
 #endif
     using fnRtlGetVersion = NTSTATUS(NTAPI*)(PRTL_OSVERSIONINFOEXW lpVersionInformation);
-    auto RtlGetVersion = (fnRtlGetVersion)GetProcAddress(GetModuleHandleW(L"ntdll.dll"), "RtlGetVersion");
-    if (RtlGetVersion)
+    const auto& ntModule{ ::GetModuleHandleW(L"ntdll.dll") };
+    if (ntModule)
     {
+        auto RtlGetVersion = (fnRtlGetVersion)GetProcAddress(ntModule, "RtlGetVersion");
```

Comment:
It's probably more important to null check the function than the dll. The dll probably exists on all machines.